client/images/RoundAvatar.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
not essential but i always run SVGs via an optimiser see here: https://jakearchibald.github.io/svgomg/

Reply:
I think they are getting a UX designer so this might change soon anyway